src/css/skins/seven.less

```diff
@@ -233,27 +228,29 @@
     .jw-icon-tooltip {
         .jw-active-option {
             background-color: @active-color;
-            color: #fff;
+             color: #424242;
+             font-weight: 600;
         }
     }
     
     /* A min-width on the volume icon to keep the width from collapsing on toggle */
     .jw-icon-volume {
-        min-width: 2.6em;
+          padding-bottom: .7em;
     }
     
-    /* For the overlay containers */
+    
     .jw-time-tip,
     .jw-menu,
     .jw-volume-tip,
-    .jw-skip {
-        border: 1px solid #333;
+         {
+        border: none;
     }
     
     /* Position and padding of slider thumbnail */
     .jw-time-tip {
-        padding: 0.2em;
+        padding: 0.3em;
         bottom: 1.3em;
+        border:0px;
```

Comment:
You can set `border: 0` instead of `border:0px. Also mind the space after the colon